APMrover2/navigation.cpp

```diff
@@ -22,7 +22,7 @@ void Rover::navigate()
 	wp_distance = get_distance(current_loc, next_WP);
 
 	if (wp_distance < 0){
-		gcs_send_text(MAV_SEVERITY_CRITICAL,"<navigate> WP error - distance < 0");
+		gcs_send_text(MAV_SEVERITY_NOTICE,"<navigate> WP error - distance < 0");
```

Comment:
Bill is completely correct.  I'll remove this code from Plane and Rover.
Thanks Bill!!